shell/imports/client/admin/login-providers.js

```diff
@@ -58,6 +61,16 @@ const idpData = function (configureCallback) {
         configureCallback("ldap");
       },
     },
+    {
+      id: "oidc",
+      label: "OpenID Connect",
+      icon: "/ldap.svg", // Or use identicons
```

Comment:
The ldap.svg graphic seems plausible for LDAP and SAML because they almost always refer to SSO for internal organizations. OpenID Connect can presumably be any OpenID Connect provider (including both internal SSO and external service providers), so I feel like we should try to get an openid.svg here, especially since OpenID has a logo.